picard/tagger.py

```diff
@@ -425,7 +435,15 @@ def event(self, event):
         return super().event(event)
 
     def _file_loaded(self, file, target=None):
-        if file is None or file.has_error():
+        self._pending_files_count -= 1
```

Comment:
Was this decrement missing?

Reply:
I've created a new variable to keep track of files being added. Not really sure if File.num_pending_files can replace this.

Reply:
Ok, no problem.